python/ray/worker.py

```diff
@@ -1635,7 +1635,7 @@ def connect(
         # are the same.
         # When using an interactive shell, there is no script directory.
         if not interactive_mode:
-            script_directory = os.path.abspath(os.path.dirname(sys.argv[0]))
```

Comment:
This is a bug and somehow not triggered before this PR.
sys.argv[0] is always `python`